config-connector/tests/testcases/environments.template

```diff
@@ -15,6 +15,8 @@
 # RANDOM_ID should not be defined in the environments file.
 BILLING_ACCOUNT_ID: ${BILLING_ACCOUNT_ID?}
 DEFAULT_PROJECT_ID: ${DEFAULT_PROJECT_ID?}
+FOLDER_ID: ${FOLDER_ID?}
 IAM_MEMBER: ${IAM_MEMBER?}
 ORG_ID: ${ORG_ID?}
+OWNED_PROJECT_ID: ${OWNED_PROJECT_ID?}
```

Comment:
Same as my earlier comment - can we reuse PROJECT_ID for multiple tests - we have at this point 4 different values to indicate project ID, can we instead use PROJECT_ID across all of them?

Reply:
Ack. I'll improve the PROJECT_ID related env vars later.